proposals/wgs/overview.md

```diff
@@ -0,0 +1,35 @@
+# Kubeflow Working Group Governance Proposal
+
+## Objective ## 
+
+1. Define the processes for forming and governing Kubeflow working groups.
+1. Push technical and project decision making down to WGs
+1. Push operational responsibilities(releases, testing, etc...) down to WGs
+
+## Background ##
+
+In 2019 the Kubeflow community approved the [Kubeflow Governance Model]. 
+A major pillar of KF's governance model is working groups that own subprojects.
+At the time, formation of WGs and the precise processes by which WGs would operate
+was delayed until KF was large enough to warrant WGs.
+
+KF has now reached a size where forming WGs is critical to sustaining and growing
+the community.
+
+This proposal aims to formalize the governance of WGs. The proposal
+is based on how Kubernetes SIGs operate. In particular, the following
+docs were used as starting points and then adopted to fit Kubeflow:
+
+  * [Lifecycle of SIGs and Working Groups](https://github.com/kubernetes/community/blob/master/sig-wg-lifecycle.md)
```

Comment:
The doc told us that we should send the mail to steering@kubernetes.io . I think we should have our own doc for it.

Reply:
These are links to the original Kubernetes docs for reference not the modified versions for Kubeflow. The modified versions are the other files in this PR.

TODO: Make this a table with links to original and revised docs.